prow/cmd/deck/runlocal

```diff
@@ -17,4 +17,5 @@ cd static
 curl "https://prow.k8s.io/data.js?var=allBuilds" > data.js
 curl "https://prow.k8s.io/tide.js?var=tideData" > tide.js
 curl "https://prow.k8s.io/plugin-help.js?var=allHelp" > plugin-help.js
+curl "https://prow.k8s.io/branding.js?var=config" > branding.js
```

Comment:
Should we change the variable name too now that it is just the branding, not the full config?
LGTM though!

Reply:
done